include/bebi.h

```diff
@@ -0,0 +1,289 @@
+
+#ifndef __BEBI_H
+#define __BEBI_H
+
+/**
+ * BEBI stands for Big Endian Big Int
+ *
+ * It is meant to be used in a small-endian environment, specifically WASI-32
```

Comment:
We may need to clarify we don't yet support wasi and that this is general over all wasm32 targets